README.md

```diff
@@ -13,7 +13,7 @@ It can be used to download or upload OpenML data such as datasets and machine le
 
 [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
-Master branch:
+Main branch:
 
 [![Build Status](https://travis-ci.org/openml/openml-python.svg?branch=master)](https://travis-ci.org/openml/openml-python)
```

Comment:
`build status` and `coverage` links should also be updated, right?

Reply:
Yes and no. We don't have travis-ci any more, so I opted to remove all of them for now.